Rakefile

```diff
@@ -85,7 +85,7 @@ task :release_beta => :unit_tests do
   puts "\nDone. Commits and tags have been created. If everything appears to be in order, hit ENTER to push."
   $stdin.gets
 
-  sh "git push origin master #{version}"
+  sh "git push origin main #{version}"
```

Comment:
I don't think we're using the Rakefile for deployments anymore. Can we remove it entirely if GitHub Actions are working?

Reply:
Maybe we keep the Rakefile until we have a successful deployment using GitHub Actions.